bentoml/_internal/store.py

```diff
@@ -129,11 +132,9 @@ def register(self, tag: t.Union[str, Tag]):
         _tag = Tag.from_taglike(tag)
 
         item_path = _tag.path()
-        if self._fs.exists(item_path):
-            raise BentoMLException(
-                f"Item '{_tag}' already exists in the store {self._fs}"
-            )
-        self._fs.makedirs(item_path)
+        is_exists = self._fs.exists(item_path)
```

Comment:
let's change this back - it is expected behavior to raise that exception, it is not allowed to re-register with the same tag